src/components/Api.js

```diff
@@ -0,0 +1,89 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _request(url, options) {
+    return fetch(url, options)
+      .then(this._checkResponse)
```

Comment:
* You haven't corrected my remark https://github.com/Andrew-W01/se_project_aroundtheus/pull/1#discussion_r1673375615 


You don't have `_checkResponse` in `App`. You have `renderResult`